sale_order_qty_change_no_recompute/tests/test_sale_order_qty_change.py

```diff
@@ -7,14 +7,32 @@ class TestSaleOrderQtyChange(SavepointCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+        cls.tax_0 = cls.env["account.tax"].create(
+            {
+                "name": "TAX 0%",
+                "amount_type": "percent",
+                "type_tax_use": "sale",
+                "amount": 0.0,
+            }
+        )
         cls.product_1 = cls.env["product.product"].create(
-            {"name": "Test Product 1", "list_price": 25.00}
+            {
+                "name": "Test Product 1",
+                "list_price": 25.00,
+                "taxes_id": [(6, 0, cls.tax_0.ids)],
+            }
         )
         cls.product_2 = cls.env["product.product"].create(
-            {"name": "Test Product 2", "list_price": 30.00}
+            {
+                "name": "Test Product 2",
+                "list_price": 30.00,
+                "taxes_id": [(6, 0, cls.tax_0.ids)],
+            }
```

Comment:
Can't you make `taxes_id` empty instead of creating a tax of 0%?

Reply:
I don't know if removing `taxes_id` completely will do it, as it might be computing it somewhere else... Could you initialize it as `False`, for example?

Reply:
`taxes_id` is not necessary.

Reply:
I am not entirely sure about that... Check the Odoo code for the field: https://github.com/odoo/odoo/blob/13.0/addons/account/models/product.py#L25
If none is set, the default will be computed from the company's tax plan, so we can end up with problems in a CI environment with different modules... Maybe it's better to explicitly mark it as `False` IMHO